model_utils/tracker.py

```diff
@@ -67,17 +88,14 @@ def init_deferred_fields(self):
         if hasattr(self.instance, '_deferred') and not self.instance._deferred:
             return
 
-        class DeferredAttributeTracker(DeferredAttribute):
-            def __get__(field, instance, owner):
-                if instance is None:
-                    return field
-                data = instance.__dict__
-                if data.get(field.field_name, field) is field:
-                    instance._deferred_fields.remove(field.field_name)
-                    value = super(DeferredAttributeTracker, field).__get__(
-                        instance, owner)
-                    self.saved_data[field.field_name] = deepcopy(value)
-                return data[field.field_name]
+        class DeferredAttributeTracker(DescriptorMixin, DeferredAttribute):
+            tracker_instance = self
+
+        class FileDescriptorTracker(DescriptorMixin, FileDescriptor):
+            tracker_instance = self
+
+            def _get_field_name(self):
```

Comment:
Not the same in fact :)
Mixin has `self.field_name`, and here we have `self.field.name` (`.` instead of `_`).